Lib/test/libregrtest/main.py

```diff
@@ -87,7 +89,11 @@ def __init__(self, ns: Namespace, _add_python_opts: bool = False):
 
         # Workers
         if ns.use_mp is None:
-            num_workers = 0  # run sequentially
+            if ns.tsan:
+                # For TSAN tests, use number of CPU of the current process.
+                num_workers = os.process_cpu_count()
```

Comment:
```
"process_cpu_count"  [attr-defined]
                    num_workers = os.process_cpu_count()
                                  ^~~~~~~~~~~~~~~~~~~~
```

@AlexWaygood 
Looks like mypy does not understand the new API from Python 3.13, do you have any recommendation for
this situation?

Reply:
See other places where we already handle this error: https://github.com/python/cpython/blob/2731913dd5234ff5ab630a3b7f1c98ad79d4d9df/Lib/test/libregrtest/main.py#L495-L497